content/zh-cn/designs/telemetry/_index.md

```diff
@@ -0,0 +1,168 @@
+---
+title: "遥测"
+weight: 80
+draft: true
+personae: [foo, bar]
+---
+
+<!-- {{< synopsis-telemetry >}} -->
+从远程环境中的传感器收集数据，并将感测的测量值供IoT解决方案的其他组件使用。
+<!--more-->
+
+## 挑战
+IoT解决方案需要可靠和安全地接收不同设备在远程环境中测量的数据，并可能使用到不同的协议。另外，一旦接收到测量数据，解决方案就需要处理和路由这些数据以供解决方案的组件来使用。
+
+## 解决方案
+IoT解决方案使用"遥测"设计，通过支持成熟的通信协议来确保在间歇性中断的网络环境下能够传送测量数据，同时提供在不同测量频率和数据量的情况下可扩展的数据接收能力，以及将数据路由给其他组件使用。
+
+以下架构图展示的"遥测"设计可以提供这样的功能。
+
+![Telemetry Design](telemetry.png)
+([PPTx](atlas-telemetry.pptx))
```

Comment:
This is no longer how the designs should link to PPTx downloads. Instead you can just link to `https://github.com/aws/iot-atlas/blob/master/content/en-us/designs/iot-atlas-patterns.pptx` using `{{< ref path="/designs/iot-atlas-patterns.pptx" lang="en" >}}`. That single PPTx in the English language directory contains all the diagrams of the site's Designs.

Reply:
The link is changed to  /designs/iot-atlas-patterns.pptx